### ISSUE TITLE
NTR: MRI: subthalamic nucleus lesion (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8912]
New term request MRI: subthalamic nucleus lesion (kalia_100602015135):

**Comment by pnrobinson** [comment]
@MickeySegal
what type of lesion?

**Comment by MickeySegal** [comment]
Many types can cause the same picture.

https://www.ncbi.nlm.nih.gov/books/NBK559127/ "Hemiballismus falls as the most severe form within the spectrum of chorea. Chorea is an involuntary movement disorder characterized by brief, sudden, spontaneous, non-stereotyped, and dance-like movements in one side of the body. The classic medical school textbook localizes the lesion to the subthalamic nucleus (STN) or the nucleus hypothalamicus/corpus Luys; however, other research also localizes the lesions in other parts of the basal ganglia and caudate nucleus"

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4401901/ "Hemiballism with leg predominance caused by contralateral subthalamic haemorrhage"

https://www.hindawi.com/journals/crim/2012/768580/ "Hemiballismus, Hyperphagia, and Behavioral Changes following Subthalamic Infarct"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000920
- label: Abnormal subthalamic nucleus morphology
- parents: [Abnormality of the diencephalon (HP:0010662)](https://hpo.jax.org/app/browse/term/HP:0010662)
- pmid: PMID:32644428
- def: Any structural anomaly of the subthalamic nucleus, a small lens-shaped structure located within the diencephalon near the junction with the midbrain. The subthalamic nucleus composes the majority of the subthalamus along with the subthalamic fasciculus, ansa lenticularis, and several other smaller structures.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8912
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
correction, the new HPO id is HP:0025737

**Comment by MickeySegal** [comment]
I see that the same occurred for several new terms, now having HP:0025737, HP:0025738 and so forth, in contrast to the numbers in the emails.

**Comment by pnrobinson** [comment]
Sorry for the inconvenience! I tried to tag all of the corrections.
